test/system/projects/group_links_test.rb

```diff
@@ -209,5 +212,56 @@ def setup
 
       assert_text 'Resource not found'
     end
+
+    test 'group member of Group B can access Group A projects as it is shared with Group A' do
+      login_as users(:user24)
+
+      namespace_group_link = namespace_group_links(:namespace_group_link10)
+
+      visit namespace_project_url(namespace_group_link.namespace.parent,
+                                  namespace_group_link.namespace.project)
+
+      assert_no_text I18n.t(:'action_policy.policy.project.read?',
+                            name: namespace_group_link.namespace.name)
+
+      assert_selector 'h1', text: namespace_group_link.namespace.name, count: 1
+      assert_selector 'p', text: namespace_group_link.namespace.description, count: 1
+    end
+
+    test 'group member of Group B cannot access Group C projects as it is not shared with Group C' do
```

Comment:
**nitpick (blocking):** Shouldn't this be written as `group member of Group B cannot access Group C projects as it is not shared with Group B`

Reply:
Updated in 038f17b9ee10a2609acd0c521ba6e415362e9ca2